frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/vm-settings-tab-state-update.ts

```diff
@@ -144,6 +146,13 @@ export const nativeK8sUpdater = ({ id, dispatch, getState, changedCommonData }:
   );
 };
 
+export const templateValidationsUpdater = (options: UpdateOptions) => {
+  const validations: TemplateValidations[] = getTemplateValidations(options);
```

Comment:
not needing `const state = getState();` in this function is strange ...

Reply:
+1

Reply:
@suomiy , I don't understand why it might have performance issues in the future. Can you elaborate ?

Reply:
@suomiy, how do you suggest to get the validations to the disk-modal without using Redux store ?

Reply:
> @suomiy , I don't understand this comment at all. Can you elaborate ?
> why the validations consider `duplicate data` ?

because they are already stored in the template

> what are the dependencies I missed?
> what performance issues you are referring to? can you give an example ?

usertemplate and templates and that is why it could be a performance issue (we could still use it if we decide to, but it is better to be careful)

Reply:
> @suomiy, how do you suggest to get the validations to the disk-modal without using Redux store ?

from the templates